### PR TITLE
feat: Calculator UI

### LINKED ISSUE #1: Create a Base for the Project

### PR DESCRIPTION
**Pull request by responilla07**
Created a base design for Calculator project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
-# tuts
```
Comment (Christiann12) [inline README.md:1]: Very nice

**Review by marwinabaquin — COMMENTED** [review]
amazing

**Review by yespanalo — COMMENTED** [review]
VERY NICEEE

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+A new Flutter project.
+
+## Getting Started
+
+This project is a starting point for a Flutter application.
+
+A few resources to get you started if this is your first Flutter project:
+
+- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
+- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
```
Comment (Christiann12) [inline README.md:12]: this is good i approve

File: README.md (hunk 1 above)
Comment (marwinabaquin) [reply README.md:1]: wow nice person

Comment (yespanalo) [reply README.md:1]: aMaiizing

**Pull request merged**